src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/View/AbstractViewHandler.Android.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Xamarin.Platform.Handlers
 {
-	public partial class AbstractViewHandler<TVirtualView, TNativeView> : IAndroidViewHandler
+	public partial class AbstractViewHandler<TVirtualView, TNativeView> : Java.Lang.Object, IAndroidViewHandler
```

Comment:
Why do you need to do this? I don't think the AbstractViewHandler should inherit from Java.Lang.Object

Reply:
To get the handler here (using the View Tag): https://github.com/xamarin/Xamarin.Forms/blob/1008a65edcf3ca8bd4ade01b5c7bf537db3076d1/src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/StepperHandlerManager.cs#L41

Probably we can achieve the same in another way.  Will take a look.

Reply:
Changed to do in another way.